lib/fastlane/actions/dotgpg_environment.rb

```diff
@@ -0,0 +1,46 @@
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    class DotgpgEnvironmentAction < Action
+      def self.run(options)
+        require 'dotgpg/environment'
+        
+        if options[:dotgpg_file]
+          dotgpg_file = options[:dotgpg_file]
+        end
+
+        raise "Dotgpg file '#{File.expand_path(dotgpg_file)}' not found".red if dotgpg_file && !File.exist?(dotgpg_file)
+
+        Helper.log.info "Reading secrets from #{dotgpg_file}"
+        Dotgpg::Environment.new(dotgpg_file).apply
+      end
+
+      def self.description
+        "Reads in production secrets set in a dotgpg file and puts them in ENV. More information at https://github.com/ConradIrwin/dotgpg"
```

Comment:
Please remove the text after the `.`. More information can be in a separate method named `def self.details`